model/project_configs.go

```diff
@@ -13,13 +13,13 @@ import (
 )
 
 type ProjectConfig struct {
-	Id         string    `yaml:"_id" bson:"_id"`
-	Project    string    `yaml:"project,omitempty" bson:"project,omitempty"`
-	CreateTime time.Time `yaml:"create_time,omitempty" bson:"create_time,omitempty"`
+	Id                  string `yaml:"_id" bson:"_id"`
+	ProjectConfigFields `yaml:",inline" bson:",inline"`
+}
 
-	// These fields can be set for the ProjectRef struct on the project page, or in the project config yaml.
```

Comment:
Shouldn't we still have a comment like this?

Reply:
Yes, was accidentally removed when I changed ProjectConfig to include the fields inline. Added back